implementations/rust/ockam/ockam_api/src/cloud/share/accept.rs

```diff
@@ -0,0 +1,85 @@
+use minicbor::{Decode, Encode};
+use serde::Serialize;
+
+use super::*;
+
+#[derive(Clone, Debug, Decode, Encode, Serialize)]
+#[cbor(map)]
+#[rustfmt::skip]
+pub struct AcceptInvite {
+    #[n(1)] pub id: String,
+}
+
+#[derive(Clone, Debug, Decode, Encode, Serialize)]
+#[cbor(map)]
+#[rustfmt::skip]
+pub struct AcceptedInvite {
+    #[n(1)] pub id: String,
+    #[n(2)] pub scope: RoleInShare,
+    #[n(3)] pub target_id: String,
+}
+
+mod node {
+    use ockam_core::api::{Request, Response};
+    use ockam_core::{self, Result};
+    use ockam_multiaddr::MultiAddr;
+    use ockam_node::Context;
+
+    use crate::cloud::CloudRequestWrapper;
+    use crate::nodes::{NodeManager, NodeManagerWorker};
+
+    use super::*;
+
+    const API_SERVICE: &str = "users";
+
+    impl NodeManager {
+        pub async fn accept_invite(
+            &self,
+            ctx: &Context,
+            req: AcceptInvite,
+            route: &MultiAddr,
+            identity_name: Option<String>,
+        ) -> Result<AcceptedInvite> {
+            Response::parse_response_body(
+                self.accept_invite_response(ctx, CloudRequestWrapper::new(req, route, identity_name))
+                    .await?
+                    .as_slice(),
+            )
+        }
+
+        pub(crate) async fn accept_invite_response(
+            &self,
+            ctx: &Context,
+            req_wrapper: CloudRequestWrapper<AcceptInvite>,
+        ) -> Result<Vec<u8>> {
+            let cloud_multiaddr = req_wrapper.multiaddr()?;
+            let req_body = req_wrapper.req;
+            let label = "accept_share";
+            let req_builder = Request::post("/v0/redeem_invite").body(req_body);
```

Comment:
Why call it `redeem` on one side and `accept` on the other?

Reply:
No principled reason - I had used `redeem` on all of the backend code, and I think I'd probably favor using that wording everywhere in a vacuum. However, `accept` feels a little more approachable in user-facing copy. WDYT?

Reply:
My personal inclination would be `accept` everywhere

Reply:
This has been changed everywhere in this repo, but not on the backend.